Digizuite/Extensions/MetadataValueServiceExtensions.cs

```diff
@@ -73,19 +76,18 @@ public static class MetadataValueServiceExtensions
             field.FieldItemId = fieldResponse.ItemId;
             field.MetafieldId = fieldResponse.MetafieldId;
             field.LabelId = fieldResponse.LabelId;
-            field.Label = fieldResponse.Label;
```

Comment:
We still need to keep `label` too.

Reply:
It is there, I messed up lines a bit, but it is there :)